src/components/Form.jsx

```diff
@@ -89,6 +90,10 @@ class Form extends React.Component {
         return this.state.errors[name] || {};
     }
 
+    getAllErrors() {
+        return this.state.errors || {};
```

Comment:
Please remove default object. Object as default in state.errors is defined in constructor.